### PR TITLE
Remove  any types in Conduit

### PR DESCRIPTION
**Pull request by sdimitris**
'Any' type arguments removed from gRPC calls.
**What kind of change does this PR introduce?** 

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [x] Refactor
- [ ] Build-related changes
- [ ] Other (please describe)

**Does this PR introduce a breaking change?** 
- [ ] Yes
- [x] No

### CODE REVIEW

File: packages/router/src/controllers/interfaces/Cookie.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  options?: {
+    httpOnly?: boolean;
+    secure?: boolean;
+    maxAge?: number;
+    signed?: boolean;
+    sameSite?: string;
+    domain?: string;
+    path: string;
+  };
+}
```
Comment (kon14) [inline packages/router/src/controllers/interfaces/Cookie.ts:13]: Also add a new line at EOF.

**Review by kon14 — CHANGES_REQUESTED** [review]

**Review by kon14 — CHANGES_REQUESTED** [review]

File: modules/sms/package.json [hunk 2]
```diff
@@ -26,7 +27,8 @@
     "@grpc/grpc-js": "^1.5.2",
     "lodash": "^4.17.15",
     "@grpc/proto-loader": "^0.5.4",
-    "twilio": "3.54.2"
+    "twilio": "3.54.2",
+    "ts-proto": "^1.112.1"
```
Comment (kon14) [inline modules/sms/package.json:31]: Should be in dev dependencies.

File: modules/email/package.json [hunk 3]
```diff
@@ -38,7 +39,8 @@
     "nodemailer": "^6.4.16",
     "nodemailer-mailgun-transport": "^2.0.0",
     "nodemailer-mandrill-transport": "^1.2.0",
-    "nodemailer-sendgrid": "^1.0.3"
+    "nodemailer-sendgrid": "^1.0.3",
+    "ts-proto": "^1.112.1"
```
Comment (kon14) [inline modules/email/package.json:43]: Should be in dev dependencies.

File: modules/email/build.sh [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+
+echo "Generating typescript code"
+protoc --plugin=./node_modules/.bin/protoc-gen-ts_proto\
+  --ts_proto_out=./src/protoTypes\
+  --ts_proto_opt=onlyTypes=true\
+  ./src/email.proto
+
+echo "Cleaning up folders"
+cp ./src/protoTypes/src/*.ts ./src/protoTypes
+rm -rf ./src/protoTypes/src/
```
Comment (kon14) [inline modules/email/build.sh:12]: Add new line at EOF.

**Review by kon14 — CHANGES_REQUESTED** [review]
Coordinate with @Michael-Vol on this.
You're both adding `ts-proto` and whatnot.

**Review by Michael-Vol — APPROVED** [review]

File: modules/chat/package.json [hunk 5]
```diff
@@ -10,9 +10,10 @@
   "scripts": {
     "start": "node dist/index.js",
     "postbuild": "copyfiles -u 1 src/**/*.proto src/*.proto src/**/*.json ./dist/",
-    "build": "rimraf dist && tsc",
+    "build": "npm run generateTypes && rimraf dist && tsc",
```
Comment (kkopanidis) [inline modules/chat/package.json:13]: move to prebuild step the generation

File: modules/email/package.json [hunk 6]
```diff
@@ -9,9 +9,10 @@
   "scripts": {
     "start": "node dist/index.js",
     "postbuild": "copyfiles -u 1 src/**/*.proto src/*.proto src/**/*.json ./dist/",
-    "build": "rimraf dist && tsc",
+    "build": "npm run generateTypes && rimraf dist && tsc",
```
Comment (kkopanidis) [inline modules/email/package.json:12]: same

File: modules/sms/package.json [hunk 7]
```diff
@@ -9,8 +9,9 @@
   "scripts": {
     "start": "node dist/index.js",
     "postbuild": "copyfiles -u 1 src/**/*.proto src/*.proto src/**/*.json ./dist/",
-    "build": "rimraf dist && tsc",
+    "build": "npm run generateTypes && rimraf dist && tsc",
```
Comment (kkopanidis) [inline modules/sms/package.json:12]: same

File: packages/core/package.json [hunk 8]
```diff
@@ -12,10 +12,11 @@
     "lint": "./node_modules/.bin/eslint .",
     "test": "mocha -r ts-node/register --timeout 10000 --exit --reporter nyan src/test/**",
     "test-with-coverage": "nyc --reporter=text mocha -r ts-node/register --timeout 10000 --exit src/test/**",
-    "build": "rimraf dist && tsc",
+    "build": "npm run generateTypes && rimraf dist && tsc",
```
Comment (kkopanidis) [inline packages/core/package.json:15]: same

**Review by kkopanidis — CHANGES_REQUESTED** [review]

**Review by kkopanidis — APPROVED** [review]

File: packages/commons/build.sh [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+echo "Generating typescript code"
+protoc --plugin=./node_modules/.bin/protoc-gen-ts_proto\
+  --ts_proto_out=./src/protoTypes\
+  --ts_proto_opt=onlyTypes=true\
+  ./src/core.proto
+
+echo "Cleaning up folders"
+cp ./src/protoTypes/src/*.ts ./src/protoTypes/
+rm -rf ./src/protoTypes/src/
+rm -f ./src/core.proto
```
Comment (kon14) [inline packages/commons/build.sh:16]: Add a new line at EOF

**Review by kon14 — CHANGES_REQUESTED** [review]
Please add new lines at EOF, otherwise legit.

**Review by kon14 — APPROVED** [review]

**Pull request merged**